Pod/Classes/ArtsyAuthentication+Private.m

```diff
@@ -11,4 +16,51 @@ - (void)callback:(ArtsyToken *)token error:(NSError *)error completion:(ArtsyAut
     }
 }
 
+- (ArtsyNetworkSuccessCallback)successfulLoginBlock:(ArtsyAuthenticationCallback)callback {
+    __weak __typeof(self) weakSelf = self;
+    return ^(NSURLRequest *request, NSHTTPURLResponse *response, id JSON) {
+        __strong __typeof(weakSelf) strongSelf = weakSelf;
+        NSString *token = JSON[ArtsyOAuthTokenKey];
+        NSString *expiryDateString = JSON[ArtsyOAuthExpiryKey];
+        ISO8601DateFormatter *dateFormatter = [[ISO8601DateFormatter alloc] init];
+        NSDate *expiryDate = [dateFormatter dateFromString:expiryDateString];
+
+        ArtsyToken *artsyToken = [[ArtsyToken alloc] initWithToken:token expirationDate:expiryDate];
+
+        [strongSelf callback:artsyToken error:nil completion:callback];
+    };
```

Comment:
Not necessarily, but since the various blocks have different class names, it should be easy to find out by checking its description in a debugger or logging it.